### PR TITLE
Wip new reading

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
@AnnJohn @jmburges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+
+## In This Unit
+
+This topic will cover how to use and navigate databases based on SQL. By the end of this topic, you’ll be able to: 
+
+* Create SQLite3 databases.
+* Create, update, select, and delete data from database tables.
+* Relate data within a given database.
+* Write SQL code in both your command line and your text editor and execute the code against a database. 
+* Write Ruby programs that talk to and save data to your databases.
```
Comment (AnnJohn) [inline README.md]: This is really good. One suggestion is to mention that we use SQLite3 in that first paragraph as well as list a couple of databases. This is mentioned in the paragraph that was taken out in the [Intro and Installation Readme](https://github.com/learn-co-curriculum/sql-intro-and-installation/commit/db7dbc69f9e6513c292f0bb657a76bea98fb377f). 

Also, this is a section in the [Database Basics Readme](https://github.com/learn-co-curriculum/database-basics) that I think can be shortened and included in this Readme somewhere. Maybe also after that first paragraph so we can explain why we choose SQLite3

screen shot 2015-10-21 at 10 54 23 am

Comment (AnnJohn) [inline README.md]: @SophieDeBenedetto @jmburges

Comment (SophieDeBenedetto) [inline README.md]: fixed.

**Comment by jmburges** [comment]
looks good, :shipit:

**Pull request merged**